resources/quotes.json

```
@@ -0,0 +1,58 @@
+[
+  {
+    "quote": "<a href=\"https://youtu.be/dQw4w9WgXcQ\">Click Me!</a>",
+    "tooltip": "Did you click yet?"
+  },
+  {
+    "quote": "A leader leads by example not by force.",
+    "tooltip": "Sun Tzu said that!"
+  },
+  {
+    "quote": "All warfare is based on deception.",
+    "tooltip": "Sun Tzu said that!"
+  },
+  {
+    "quote": "Avidly exploring a more thrilling side to coding!",
+    "tooltip": "Nayaab, 2020"
+  },
+  {
+    "quote": "BTW I use arch.",
+    "tooltip": null
+  },
+  {
+    "quote": "Don't fear the OOP!",
+    "tooltip": null
+  },
+  {
+    "quote": "Introducing weekly TEJ contests!",
+    "tooltip": "ENG2D and ENG3U contests are next!"
+  },
+  {
+    "quote": "Opportunities multiply as they are seized.",
+    "tooltip": "Sun Tzu said that!"
+  },
+  {
+    "quote": "Powered by PHP!",
+    "tooltip": null
+  },
+  {
+    "quote": "Pretend inferiority and encourage his arrogance.",
+    "tooltip": "Sun Tzu said that!"
+  },
+  {
+    "quote": "Real programmers count from zero.",
+    "tooltip": "Unless you use turing."
+  },
+  {
+    "quote": "To know your Enemy, you must become your Enemy.",
+    "tooltip": "Sun Tzu said that!"
+  },
+  {
+    "quote": "Turing > Ruby > C > Java > C++ > Python",
+    "tooltip": null
```

```suggestion
    "tooltip": "C > Python > Ruby > Java > C++ > Turing"
```
That way we can be even more contentious!